src/overlays/actors/ovl_Obj_Makeoshihiki/z_obj_makeoshihiki.c

```diff
@@ -7,7 +7,6 @@
 void ObjMakeoshihiki_Init(Actor* thisx, GlobalContext* globalCtx);
```

Comment:
Could you add the file description?
For example: https://github.com/zeldaret/mm/blob/9eb438860cf1bcfe3ee31b1f50c43114606d4f79/src/overlays/actors/ovl_En_Ma4/z_en_ma4.c#L1